composer.json

```
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
+        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || 3.0.0-beta2",
         "doctrine/phpcr-bundle": "^1.3 || ^2.0.0",
```

ah sorry, wrong bundle :facepalm: 

```suggestion
        "doctrine/phpcr-bundle": "^1.3 || ^2.0.0 || ^3.0@beta",
```